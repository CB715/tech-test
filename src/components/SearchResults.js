import React from "react";

import '../styles/searchResults.css'

const SearchResults = ({ results }) => {
  if (!results.length) {
    return <p className='search-results__no-results'>No results</p>
  } else {
    return (
      <div className='search-results'>
        <div className='search-results__text'>
          <p>Search Results:</p>
        </div>
        <div className='search-results__images'>
        {results.map((image) => (
          <img className="card-image" src={image} alt="spaceImage" />
        ))}
        </div>
      </div>
    )
  };
};

export default SearchResults
