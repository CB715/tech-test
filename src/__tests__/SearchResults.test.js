import React from 'react';
import { render } from '@testing-library/react'
import SearchResults from "../components/Search";

describe('Search Results', () => {
  const { asFragment } = render(<SearchResults />);

    it('renders correctly', () => {
      expect(asFragment()).toMatchSnapshot();
  });
});