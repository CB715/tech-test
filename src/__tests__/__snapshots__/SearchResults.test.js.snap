// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Results renders correctly 1`] = `
<DocumentFragment>
  <p
    class="search-results__no-results"
  >
    No results
  </p>
</DocumentFragment>
`;
